src/app/screens/ledger/importLedgerAccount/index.tsx

```diff
@@ -475,55 +215,55 @@ function ImportLedger(): JSX.Element {
     return { btcCreds, ordinalsCreds, newAccountId };
   };
 
-  const importStxAccounts = async () => {
+  const importStxAccounts = async (showAddress: boolean) => {
+    setIsButtonDisabled(true);
     const transport = await Transport.create();
-    const newAccountId = getNewAccountId();
-    setAccountId(newAccountId);
-    const deviceNewAccountIndex = getDeviceNewAccountIndex(masterPubKey);
-    const { address, publicKey } = await importStacksAccountFromLedger(
-      transport,
-      network.type,
-      0,
-      deviceNewAccountIndex,
-    );
-    setStacksCredentials({
-      address,
-      publicKey,
-    });
-    await transport.close();
-  };
 
-  const handleSkipToEnd = () => {
-    setCurrentStepIndex(6);
+    try {
+      const stacksCreds = await importStacksAccountFromLedger(
+        transport,
+        network.type,
+        0,
+        accountId,
```

Comment:
what is accountId here?

Reply:
The `accountId` is set in the `importBtcAccounts` function. However, I just updated this line to use `deviceAccountIndex` as it's the appropriate value to pass here